### PR TITLE
LPS-131305 add support for scoped entities + LPS-138882 rename paths

### PR DESCRIPTION
**Pull request by nhpatt**
This add endpoints like /o/c/scopes/{scopeId}. I'm not using `/object-entries` as we do not want any reference to it in the paths/OpenAPI and we want to expose only the user chosen name.

This methods are disabled when the `ObjectDefinition` is scoped by company and can not be used via REST, GraphQL nor exposed in OpenAPI. The opposite is also true, when the `ObjectDefinition` is scoped by group/asset library, the general unscoped methods disappear.

I'm using scope instead of siteId to allow other types of scopes that are planned for the future. I'm using a string to be able to use everything, like the group/asset library name or the id (both will work).

I'm also renaming the APIs to use `/o/c/my-element` instead of `/o/c-my-element`. It also adds namespaces in GraphQL to be able to use queries like:

```
mutation {
  c {
    createMyElement(...) {
     ...
    }
  }
}
```

It's probable that some poshi tests that depends on the path will fail, let's see :)

### CODE REVIEW

File: modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/deployer/ObjectDefinitionDeployerImpl.java [hunk 1]
```diff
@@ -153,6 +170,56 @@
 		return serviceRegistrations;
 	}
 
+	private void _excludeScopedMethods(
```
Comment (javierdearcos) [inline modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/deployer/ObjectDefinitionDeployerImpl.java]: Smart and clean way to deactivate the paths that don't apply

**Review by javierdearcos — APPROVED** [review]
Nice 🙂 

Feels good to get rid of the c_ in the paths

**Comment by javierdearcos** [comment]
I've resolved a conflict from the GitHub web editor and have a really ugly commit name...

**Comment by javierdearcos** [comment]
ci:forward

**Comment by javierdearcos** [comment]
😞  The automatic name does not start with the LPS and makes the SF fails...

Sorry... I want it to help to solve the conflict and forward it but it was worse

**Comment by nhpatt** [comment]
ci:forward

**Comment by nhpatt** [comment]
ci:test:sf

**Comment by nhpatt** [comment]
This was sent and iterated directly with Brian here: https://github.com/brianchandotcom/liferay-portal/pull/106988

**Pull request closed without merging**